.github/workflows/unit-integration-tests.yml

```diff
@@ -245,18 +245,26 @@ jobs:
       - name: Testing outlier gRPC v1.2
         run: |
               # This hacky approach is necessary because running the tests regularly
-              # do not allow using grpc-go@v1.2.0 alongside sketches-go@v1.0.0
+              # do not allow using grpc-go@v1.2.0 alongside sketches-go@v1.1.0.
```

Comment:
Shouldn't this read "do not allow using grpc-go@v1.2.0 alongside sketches-go@v1.0.0" instead?
1.0.0 vs 1.1.0 — package `github.com/DataDog/sketches-go/ddsketch/encoding` is avail as of and including 1.1.0, but not prior (1.0.0)

Reply:
I changed it because it was aligned with the version checked later in the script. It's not possible to test `grpc-go@v1.2.0` with `sketches-go@v1.0.0` as our code expectes `sketches-go` to contain the package described.